packages/dds/ordered-collection/src/interfaces.ts

```diff
@@ -108,6 +108,8 @@ export interface IConsensusOrderedCollectionEvents<T> extends ISharedObjectEvent
  * @legacy
  * @alpha
  */
+
```

Comment:
Not sure if this extra blank line would break the connection between the TSDoc block above and the interface?

Reply:
I don't believe it does. TSDoc comments are associated with the first valid code that follows them, regardless of blank lines or directives like eslint-disable. You can verify this by hovering over the interface showing the related comment.

Reply:
Still probably best to remove the blank line though